decidim-elections/app/controllers/decidim/elections/admin/trustees_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    module Admin
+      # This controller allows the create or update trustees.
+      class TrusteesController < Admin::ApplicationController
+        helper_method :trustees, :trustee
+
+        def new
+          enforce_permission_to :create, :trustee
+          @form = form(TrusteeForm).instance
+        end
+
+        def create
+          enforce_permission_to :create, :trustee
+          @form = form(TrusteeForm).from_params(params)
+
+          AddUserAsTrustee.call(@form, current_user) do
+            on(:ok) do
+              flash[:notice] = I18n.t("trustees.create.success", scope: "decidim.elections.admin")
+              redirect_to trustees_path
+            end
+
+            on(:invalid) do
+              flash.now[:alert] = I18n.t("trustees.create.invalid", scope: "decidim.elections.admin")
+              render :new
+            end
+
+            on(:exists) do
+              flash.now[:alert] = I18n.t("trustees.create.exists", scope: "decidim.elections.admin")
+              render :new
+            end
+          end
+        end
+
+        def edit
```

Comment:
I think the `:edit` action is not needed since we are only updating the "considered" in the `TrusteesParticipatorySpacesController`. So this action can be removed (with the related views)